mobs/nakki_v2/ai_states/NakkiEmergeOnTriggerState.cs

```diff
@@ -0,0 +1,93 @@
+using Godot;
+using System;
+using System.Linq;
+
+using Jakojaannos.WisperingWoods.Util.Editor;
+
+namespace Jakojaannos.WisperingWoods;
+
+[Tool]
+public partial class NakkiEmergeOnTriggerState : NakkiAiState {
+	public override string[] _GetConfigurationWarnings() {
+		return (base._GetConfigurationWarnings() ?? Array.Empty<string>())
+			.Union(this.CheckCommonConfigurationWarnings())
+			.ToArray();
+	}
+
+	[Export]
+	[MustSetInEditor]
```

Comment:
I usually move the `[MustSetInEditor]` properties as the last exported members of a class, and add a `[ExportGroup("Prewire")]` to the first `[MustSetInEditor]` proprety.

This creates a collapsible "Prewire" group in the inspector, neatly packing all "set-once-and-forget" `MustSetInEditor` properties there. I think the group is additionally collapsed by default, so that really helps keep the inspector clean.